libswirl/types.h

```diff
@@ -360,8 +360,7 @@ using namespace std;
 #endif
 
 
-// NOTE: Always inline for macOS builds or it causes duplicate symbol linker errors
-#if DEBUG && HOST_OS != OS_DARWIN
+#if DEBUG
```

Comment:
Can you revert this?  it might break the osx build

Reply:
sure thing! i didn't even notice that change.. its fixed now i suppose! :D